_data/monthly-calls.yml

```diff
@@ -1,6 +1,19 @@
 videos:
+  - title: Introduction to Web Components
+    subtitle: Learn the basics about Web Components and what this technology means for USWDS
+    description: |
+     Dan Williams and James Mejia from USWDS described the basics of Web Components: what they are, how this technology works, and how Web Components can simplify how teams implement and stay up-to-date with the design system.
+    date: April 2024
+    id: GHomUbYTgwQ
+    event_link: https://digital.gov/event/2024/04/18/uswds-monthly-call-april-2024/
+    missing_script: true
```

Comment:
It looks like the event page has the script, so we should be able to remove the `missing_script` data key. This will update the text on the page to say "{{ video.date }} script and more at Digital.gov" instead of "{{ video.date }} event page at Digital.gov". 

We can probably do the same for March 2024. 
```suggestion
```

Reply:
Removed `missing_script` data key from all months that have scripts on their event pages in [0677e1c](https://github.com/uswds/uswds-site/pull/2671/commits/0677e1c8045a71384b60c15b98e3fe407c0e2108)